### DISCUSSION TITLE
terminate() and releasing memory

### DISCUSSION THREAD

**Discussion by CADBIMDeveloper** [elringus/bootsharp#117] [Q&A] [answered]
Hi @Elringus ,

I'm not sure, if it's a bug or intended behavior... And also if this issue is related to DotNetJS or .Net to WASM compilation. I'm curious how can I free memory after I don't need it.

I tried
```ts
await dotnet.terminate()
```
But seems, that didn't help:
![screenshot](https://user-images.githubusercontent.com/14212214/221936516-13f85c0f-5ed8-4358-b74e-391def572487.png)
![screenshot](https://user-images.githubusercontent.com/14212214/221937106-2d5edd81-2fc5-4dbb-a561-33e29b783788.png)

**Comment by elringus** [comment, marked as answer]
Hey, It's a known issue, see #22 

TL;DR: Should be fixed after #20 is solved.

**Reply by CADBIMDeveloper** [reply]
Thanks!
